AntiCovid/app/src/main/java/com/swtug/anticovid/view/statistics/StatisticFragment.kt

```diff
@@ -43,40 +65,92 @@ class StatisticFragment : BaseFragment() {
+        ).also { adapter ->
+            // Specify the layout to use when the list of choices appears
+            adapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item)
+            // Apply the adapter to the spinner
+            spinnerCountries.adapter = adapter
+            spinnerCountries.setSelection(AUSTRIA)
+
+            totalCases = view.findViewById<TextView>(R.id.totalcasesvalue)
+            totalTest = view.findViewById<TextView>(R.id.totaltestedvalue)
+            totalDeath = view.findViewById<TextView>(R.id.totaldeathvalue)
+            todayCases = view.findViewById<TextView>(R.id.casestodayvalue)
+            todayDeath = view.findViewById<TextView>(R.id.deathtodayvalue)
+            todayRecovered = view.findViewById<TextView>(R.id.recoveredtodayvalue)
+            totalRecovered = view.findViewById<TextView>(R.id.totalrecoveredvalue)
+            population = view.findViewById<TextView>(R.id.populationvalue)
+            update = view.findViewById<TextView>(R.id.lastupdatevalue)
+            imvFlag = view.findViewById<ImageView>(R.id.imv_Showflag)
+            active = view.findViewById<TextView>(R.id.activevalue)
+        }
+    }
+
+    private fun updateData(view: View?, _countryList: List<MyCountry>, _countryID: Int){
+        active.text = _countryList[_countryID].active.toString()
+        totalCases.text = _countryList[_countryID].cases.toString()
+        totalTest.text = _countryList[_countryID].tests.toString()
+        totalDeath.text = _countryList[_countryID].deaths.toString()
+        todayCases.text = _countryList[_countryID].todayCases.toString()
+        todayDeath.text = _countryList[_countryID].todayDeaths.toString()
+        todayRecovered.text = _countryList[_countryID].todayRecovered.toString()
+        totalRecovered.text = _countryList[_countryID].recovered.toString()
+        population.text = _countryList[_countryID].population.toString()
+        val date = Date(_countryList[_countryID].updated)
+        val format = SimpleDateFormat("dd.MM.yyyy - HH:mm")
+        update.text = format.format(date)
+        Picasso.get().load(_countryList[_countryID].countryInfo.flag).into(imvFlag);
+
+    }
+
+    private fun setErrorMessage(){
+        active.text = R.string.statistic_error_message.toString()
+        totalCases.text = R.string.statistic_error_message.toString()
+        totalTest.text = R.string.statistic_error_message.toString()
+        totalDeath.text = R.string.statistic_error_message.toString()
+        todayCases.text = R.string.statistic_error_message.toString()
+        todayDeath.text = R.string.statistic_error_message.toString()
+        todayRecovered.text = R.string.statistic_error_message.toString()
+        totalRecovered.text = R.string.statistic_error_message.toString()
+        population.text = R.string.statistic_error_message.toString()
+        update.text = R.string.statistic_error_message.toString()
```

Comment:
`R.string.<ResourceName> ` returns an Integer. So your error message is a number. user `getString(R.string.<ResourceName>)` instead